LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2013 Despo Pentara
```

Comment:
Any code that I've contributed here is owned by my employer, which requires attribution as part of our open-source contributions policy. I have to list the company somewhere.